packages/lib/src/components/internal/SecuredFields/binLookup/triggerBinLookUp.ts

```diff
@@ -34,6 +38,49 @@ export default parent => {
             ).then((data: BinLookupResponseRaw) => {
                 // If response is the one we were waiting for...
                 if (data?.requestId === currentRequestId) {
+                    if (process.env.NODE_ENV === 'development') {
```

Comment:
Is it intended to leave this code-block here? Seems like it was used for dev purposes

Reply:
Yeah, it is deliberate.
It's a way, when you're developing, to intercept and mock the binLookup response.
I'm continually having to copy & paste this block of code into `triggerBinLookup` - so I thought I'd leave it there, to only be included when building in dev mode, so devs can easily test binLookup. For a production build it will be excluded